src/Commands/Multidev/MergeToDevCommand.php

```diff
@@ -15,20 +15,20 @@ class MergeToDevCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Merge the changes in a multidev environment into the dev environment
+     * Merges code commits from a Multidev environment into the Dev environment.
      *
      * @authorize
      *
      * @command multidev:merge-to-dev
      * @aliases env:merge-to-dev
      *
-     * @param string $site_env Site & environment to merge into dev, in the form `site-name.env`.
-     * @option boolean $updatedb True to update the database along with this merge
+     * @param string $site_env Site & Multidev environment in the form `site-name.env`
+     * @option boolean $updatedb Run update.php afterwards
      *
-     * @usage terminus multidev:merge-to-dev <site>.<env>
-     *   Merges the code from the multidev <env> environment of <site> into its dev environment
+     * @usage terminus multidev:merge-to-dev <site>.<multidev>
+     *     Merges code commits from <site>'s <multidev> environment into the Dev environment.
      * @usage terminus multidev:merge-to-dev awesome-site.multidev-env --updatedb
```

Comment:
The names of the targets on this line never got updated. I'm sorry and worried I didn't catch this until the third pass.